pages/glossary/programatically-determined-link-context.md

```diff
@@ -0,0 +1,19 @@
+--
+title: Programmatically Determined Link Context
+key: programmatically-determined-link-context
+--
+
+In HTML, information that is programmatically determinable from a link includes [text nodes][] that are [included in the accessibility tree][] with a non-empty ("") [accessible name][] and are
```

Comment:
I don't think this should be limited to text nodes. That way you're ignoring images and other things with accessible names through attributes.